incubator/hnc/e2e/tests/delete_crd_test.go

```diff
@@ -62,4 +64,18 @@ var _ = Describe("Delete-anchor-crd", func() {
 		// verify
 		mustRun("kubectl get -oyaml rolebinding foo -n", nsChild)
 	})
+
+	It("should fully delete all CRDs", func() {
+		// set up
+		mustRun("kubectl create ns", nsParent)
+		mustRun("kubectl hns create", nsChild, "-n", nsParent)
+		// test
+		mustRun("kubectl delete crd hierarchyconfigurations.hnc.x-k8s.io")
+		time.Sleep(1 * time.Second)
```

Comment:
What's this one for?

Reply:
It's from the shell script, the original comment is "Delete the HC CRD, then wait 1s, then delete the anchor CRD." My guess is that `delete crd hierarchyconfigurations.hnc.x-k8s.io` takes some time

Reply:
Maybe try removing it in a followup CL. Also if you see stuff like this that you don't understand, it's good to comment it or see if you can delete it.